servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/RetryingHttpRequesterFilter.java

```diff
@@ -642,6 +672,22 @@ public interface DelayedRetry {
         Duration delay();
     }
 
+    /**
+     * Callbacks invoked on a retry attempt.
+     */
+    @FunctionalInterface
+    public interface RetryCallbacks {
```

Comment:
Are you envisioning more methods to be added here in the future since the name is plural?